src/Components/Auth/Profile/Profile.jsx

```diff
@@ -0,0 +1,69 @@
+import axios from "axios";
+import { useEffect } from "react";
+import { useAuth } from "../../../Context";
+import { VIZZ_API } from "../../../utils";
+import { FaUserCircle } from "react-icons/fa";
+import "./styles.css";
+
+export const Profile = () => {
+	const {
+		state: { userDetails, token },
+		dispatch,
+		logout,
+	} = useAuth();
+
+	useEffect(() => {
+		if (!userDetails) {
+			(async () => {
+				try {
+					const {
+						data: { response },
+						status,
+					} = await axios({
+						url: `${VIZZ_API}/users/info`,
+						method: "GET",
+						headers: {
+							Authorization: `Bearer ${token}`,
+						},
+					});
+					console.log({ response });
+
+					if (status === 200) {
+						dispatch({
+							type: "SET_USER_DETAILS",
+							payload: { userDetails: response },
+						});
+					}
+				} catch (error) {
+					console.log(error);
```

Comment:
No error handling in the UI? I couldn't simulate it in the UI as there's no error as of now. 😅😶‍🌫️